demo5/ReadMe.txt

```diff
@@ -0,0 +1,23 @@
+Demonstration of using message queues together with the `selectors` module in Python.
+
+The `selectors` module provides high-level I/O multiplexing akin to an event
```

Comment:
It would be good to mention that this is a standard library module, and also provide a link to the doc (https://docs.python.org/3/library/selectors.html)